### DISCUSSION TITLE
Getting blocked by ebay

### DISCUSSION THREAD

**Discussion by mhmuc1** [Second-Hand-Friends/kleinanzeigen-bot#201] [Q&A]
Hi,

my account is getting blocked every time I start using the bot. 
I am using the pre-compiled windows version.

Is it the fixed sleep time that is getting recognized by ebay?
Maybe I should compile my own version to randomize that?

Is anyone else having issues like that?
Thanks for your help.

**Comment by sebthom** [comment]
Try to use a non-private browser window in the config.yaml.
```yaml
browser:
  use_private_window: false
```

**Comment by mhmuc1** [comment]
Hi,
thank you for the answer. I was already using this feature.
I will try to use chrome instead of edge
